gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/sampling/RandomEdgeSampling.java

```diff
@@ -79,6 +79,7 @@ public LogicalGraph sample(LogicalGraph graph) {
       .distinct(new Id<>());
 
     DataSet<Vertex> newVertices = newSourceVertices.union(newTargetVertices).distinct(new Id<>());
-    return graph.getConfig().getLogicalGraphFactory().fromDataSets(newVertices, newEdges);
+
+    return graph.getFactory().fromDataSets(newVertices, newEdges);
```

Comment:
I write it down here but that should be done for all sampling algorithms: Please add a note in the Sampling JavaDoc that a new logical graph will be returned containing the sampled graph elements.